app/views/candidate_mailer/fraud_match_email.text.erb

```diff
@@ -1,11 +1,7 @@
 Dear <%= @application_form.first_name %>,
 
-We’ve noticed that you’ve started multiple applications on the DfE’s Apply for teacher training.
+You’ve created more than one account on Apply for teacher training.
 
-Submitting applications for more than 3 courses at a time is not allowed, as stated in our terms of use.
+As you have already submitted an application, the account with the unsubmitted application will be locked.
```

Comment:
This should only show if they have submitted an application already, I think? ie we'll need an `if` in here.